documentation/MediaFeatureArticle.json

```diff
@@ -1,145 +1,141 @@
+    "mainTitle" : "9Z2RQeaWFy",
     "alternativeTitles" : {
-      "nn" : "srOzjdzeoPOnIK"
+      "sv" : "BUovckAxDcavPb"
     },
-    "language" : "http://lexvo.org/id/iso639-3/pol",
+    "language" : "http://lexvo.org/id/iso639-3/por",
     "date" : {
       "type" : "PublicationDate",
-      "year" : "EThSmfkHab",
-      "month" : "ARhCLnlvZBl3V",
-      "day" : "zmuaylPMqa4WR69M"
+      "year" : "XIgx1QEZyDdyNLZz2X5",
+      "month" : "P0UhKnIQGLm8XbEEZjF",
+      "day" : "yrAoyywQNKJU"
     },
     "contributors" : [ {
       "type" : "Contributor",
       "identity" : {
         "type" : "Identity",
-        "id" : "https://www.example.com/AbuPebB4kDPudv6E",
-        "name" : "UX3DrHwqQQieT",
+        "id" : "https://www.example.com/zNVzLy6Ntv",
+        "name" : "AQorY1Lx3McGlLj",
         "nameType" : "Personal",
-        "orcId" : "8Gh7K3VZ9Ki"
+        "orcId" : "PP1XXs8QWCNYnrXarL"
       },
       "affiliations" : [ {
         "type" : "Organization",
-        "id" : "https://www.example.com/xkYKUsQG4YmOhS",
+        "id" : "https://www.example.com/a9Qmj12Z5G3Xxg",
         "labels" : {
-          "vEy4vpgVByovb3w" : "JrJ5YnAx19TVpDGcBn"
+          "LnxYZF624tbRcqsr" : "V1kOJNzYOX6vFl"
         }
       } ],
-      "role" : "Designer",
-      "sequence" : 6,
+      "role" : "Organizer",
+      "sequence" : 4,
       "correspondingAuthor" : false
     }, {
       "type" : "Contributor",
       "identity" : {
         "type" : "Identity",
-        "id" : "https://www.example.com/w9I6U29qQB",
-        "name" : "UFOfTXzD9YI8gL",
+        "id" : "https://www.example.com/MsVmqQrS5c085h9Q8",
+        "name" : "7WN0IXIRLk",
         "nameType" : "Organizational",
-        "orcId" : "yh1U4c4AGLUN7svyaB"
+        "orcId" : "wj7YTuHviaEx"
       },
       "affiliations" : [ {
         "type" : "Organization",
-        "id" : "https://www.example.com/mtoOgOSrQhA57n",
+        "id" : "https://www.example.com/viv4ZRTgQrmh0pCf",
         "labels" : {
-          "ZNBHGZhJ11" : "qh0rzDxFO8Ejf0gde6"
+          "NBTjA7gmc3qfS7Ka04O" : "gtNMyMqLCmXXWfTIKk"
         }
       } ],
-      "role" : "DataCurator",
-      "sequence" : 8,
+      "role" : "Director",
+      "sequence" : 7,
       "correspondingAuthor" : false
     } ],
-    "npiSubjectHeading" : "mPmcl2mUjkVl",
-    "tags" : [ "YG5jxbvOmX" ],
-    "description" : "u95IRIfXbTZp89YWLV",
+    "npiSubjectHeading" : "sT5TCm4EyfJzBCwNMCL",
+    "tags" : [ "qHhhKAjLaErTKFosq" ],
+    "description" : "t5EDiIP6MSgFh9s9Z",
     "reference" : {
       "type" : "Reference",
       "publicationContext" : {
-        "type" : "MediaContribution",
-        "medium" : {
-          "type" : "TV"
-        },
-        "format" : "Video",
-        "disseminationChannel" : "qZCKgb52rONtspX",
-        "partOf" : {
-          "type" : "SeriesEpisode",
-          "series" : "JF6pV2KqY0",
-          "seriesPart" : "WSLQLHg0pcKkb6kX"
-        }
+        "type" : "MediaContributionPeriodical",
```

Comment:
Vanskelig å få helt oversikt når det er diff i så mange filer, men tldr så tolker jeg det som at det som før bare var `publicationContext.type: "MediaContributition"` nå blir `publicationContext.type: "MediaContributition" | "MediaContributionPeriodical"`.
I så fall helt likt måten det virker for Journal i dag, så da mener jeg vi skal få håndtert det i frontend også uten alt for mye triksing